apps/dao/src/components/ConnectWallet.tsx

```diff
@@ -0,0 +1,31 @@
+import { isLoading } from '@/ui/utils'
+import useStore from '@/store/useStore'
+
+import ConnectWalletPrompt from '@/ui/ConnectWalletPrompt'
+
+type ConnectWalletProps = {
+  description: string
+  connectText: string
+  loadingText: string
+}
+
+const ConnectWallet: React.FC<ConnectWalletProps> = ({ description, connectText, loadingText }) => {
+  const updateConnectWalletStateKeys = useStore((state) => state.wallet.updateConnectWalletStateKeys)
+  const connectState = useStore((state) => state.connectState)
+  const theme = useStore((state) => state.themeType)
+
+  const loading = isLoading(connectState)
+
+  return (
+    <ConnectWalletPrompt
+      connectWallet={updateConnectWalletStateKeys}
+      description={description}
+      connectText={connectText}
+      loadingText={loadingText}
+      isLoading={loading}
+      theme={theme === 'dark' ? 'dark' : 'light'}
```

Comment:
What happened to chad?

Reply:
dark and light refers to the two different variants of the background image, light and chad shares the same